main/webservices/registration.soap.php

```diff
@@ -2014,7 +2013,6 @@ function WSEditUserWithPicture($params)
     $user
         ->setEmail($email)
         ->setStatus($status)
-        ->setOfficialCode($official_code)
```

Comment:
That's what I did first and then I saw that there is no official_code in the parameters passed to this webservice (tns:editUserWithPicture), so what is the point of adding a check ?

The other option is to add this variable to the parameters, but if I change the declaration of the parameters, it might not work with other system that are using this webservice without this parameter. That's why I decided to delete it.